core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsBackupResponse.java

```diff
@@ -90,15 +98,11 @@ public void waitForBackupToFinish() throws Exception {
             elapsedTime += timeService.timeDuration(startNanos, MILLISECONDS);
          }
 
-         if (response instanceof ExceptionResponse) {
-            Throwable remoteException = ((ExceptionResponse) response).getException();
-            log.tracef(remoteException, "Got error backup response from site %s", siteName);
-            errors.put(siteName, remoteException);
-         } else {
-            log.tracef("Received response from site %s: %s", siteName, response);
-         }
+         log.tracef("Received response from site %s: %s", siteName, response);
+      }
+      if (hasSyncBackups) {
+         timeElapsedConsumer.accept(timeService.timeDuration(sendTimeNanos, MILLISECONDS));
```

Comment:
why is this sendTimeNanos and then we pass milliseconds ?

Reply:
The `timeElapsedConsumer` contract is the amount of time spent to wait for the request to complete in milliseconds.
The `timeDuration()` receives the start timestamp (in nanoseconds) and returns in the duration in desired `TimeUnit` (in this case milliseconds)

Reply:
ok!!